src/app/modules/item/pages/item-current-situation/item-current-situation.component.html

```diff
@@ -0,0 +1,3 @@
+<h1>User situation</h1>
+<alg-selection (change)="onViewChanged($event)" [items]="viewItems" [selected]="viewSelected"></alg-selection>
```

Comment:
Arf, the items parameter is still a any type :-/  (from the design)
This is the kind of things I try to fix directly when I see it. I'll fix this alg-selection when we'll be done with this PR...